src/search/merge_and_shrink/merge_scoring_function_total_order.cc

```diff
@@ -202,35 +181,58 @@ void MergeScoringFunctionTotalOrder::add_options_to_parser(
-            "https://ai.dmi.unibas.ch/papers/sievers-et-al-icaps2016.pdf",
-            "Proceedings of the 26th International Conference on Automated "
-            "Planning and Scheduling (ICAPS 2016)",
-            "294-298",
-            "AAAI Press",
-            "2016") +
-        "Furthermore, using the atomic_ts_order option, this scoring function, "
-        "if used alone in a score based filtering merge selector, can be used "
-        "to emulate the corresponding (precomputed) linear merge strategies "
-        "reverse level/level (independently of the other options).");
-    MergeScoringFunctionTotalOrder::add_options_to_parser(parser);
-
-    options::Options options = parser.parse();
+static shared_ptr<MergeScoringFunction>_parse(plugins::OptionParser &parser) {
+    {
+        parser.document_synopsis(
+            "Total order",
+            "This scoring function computes a total order on the merge candidates, "
+            "based on the specified options. The score for each merge candidate "
+            "correponds to its position in the order. This scoring function is "
+            "mainly intended as tie-breaking, and has been introduced in the "
+            "following paper:"
+            + utils::format_conference_reference(
+                {"Silvan Sievers", "Martin Wehrle", "Malte Helmert"},
+                "An Analysis of Merge Strategies for Merge-and-Shrink Heuristics",
+                "https://ai.dmi.unibas.ch/papers/sievers-et-al-icaps2016.pdf",
+                "Proceedings of the 26th International Conference on Automated "
+                "Planning and Scheduling (ICAPS 2016)",
+                "294-298",
+                "AAAI Press",
+                "2016") +
+            "Furthermore, using the atomic_ts_order option, this scoring function, "
+            "if used alone in a score based filtering merge selector, can be used "
+            "to emulate the corresponding (precomputed) linear merge strategies "
+            "reverse level/level (independently of the other options).");
+        MergeScoringFunctionTotalOrder::add_options_to_parser(parser);
+    }
+    plugins::Options options = parser.parse();
     if (parser.dry_run())
         return nullptr;
     else
         return make_shared<MergeScoringFunctionTotalOrder>(options);
 }
 
-static options::Plugin<MergeScoringFunction> _plugin("total_order", _parse);
+static plugins::Plugin<MergeScoringFunction> _plugin("total_order", _parse);
+
+static vector<pair<string, string>> _enum_data_atomic_ts_order() {
+    return {
+        {"reverse_level",
+         "the variable order of Fast Downward"},
+        {"level",
+         "opposite of reverse_level"},
+        {"random",
+         "a randomized order"}
+    };
+}
+
+static vector<pair<string, string>> _enum_data_product_ts_order() {
+    return {
+        {"old_to_new",
+         "consider composite transition systems from oldest to most recent"},
```

Comment:
This previously was "from most recent to oldest". Given that the new version makes more sense, was the old one a typo?

Reply:
yes, this was a mistake in the old documentation